src/js/common/schemaform/FormApp.jsx

```diff
@@ -107,7 +107,11 @@ class FormApp extends React.Component {
       && status !== this.props.loadedStatus
       && !window.location.pathname.endsWith('/error')
     ) {
-      newProps.router.push(`${newProps.formConfig.urlPrefix || ''}error`);
+      let action = 'push';
+      if (window.location.pathname.endsWith('resume')) {
+        action = 'replace';
+      }
+      newProps.router[action](`${newProps.formConfig.urlPrefix || ''}error`);
```

Comment:
I found that if you're on the error page and you came from a content page or the account page, clicking Back sends you back to the resume page, and puts you in a loop. This replaces the resume entry in the history and avoids that.